SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py

```diff
@@ -90,145 +88,8 @@ def customiseRun2EraExtras(process):
         process.simCsctfTrackDigis.SectorReceiverInput = cms.untracked.InputTag("simCscTriggerPrimitiveDigis", "MPCSORTED")
 
     # deal with L1 Emulation separately:
-    # replace the L1 menu from the global tag with one of the following alternatives
```

Comment:
Note that here I've replaced the copy of the L1 customisations with a call to the original customisation function: the copy had become out-of-sync from the original, and I hardly see the point of maintaining the same customisation in two places.